src/Cdn/custom/Remove-AzCdnProfile.ps1

```diff
@@ -155,6 +155,10 @@ function Remove-AzCdnProfile {
     )
     
     process {
+        $hasAsJob = $PSBoundParameters.Remove('AsJob')
```

Comment:
It is not a good idea to remove and add back parameters.
I propose to copy the $PSBoundParameters to a new variable for Get-AzCdnProfile

Reply:
Copy. Thanks for you advice. Will fix.